### DISCUSSION TITLE
Welcome to flutter_slidable Discussions!

### DISCUSSION THREAD

**Discussion by letsar** [letsar/flutter_slidable#187] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of flutter_slidable. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by puhgeh** [comment]
Hi!

I think this is a great package! Is there a way that it clips only to the boundary of the ListView when dragged?

![Screen Shot 2021-08-03 at 5 00 27 PM](https://user-images.githubusercontent.com/5802406/127988995-4bdddb5e-30fc-4569-962a-49c9709bed24.png)

**Reply by letsar** [reply]
Did you use the 1.0.0 version? (https://pub.dev/packages/flutter_slidable/versions/1.0.0-dev.9)
It should be fixed on that version

**Reply by puhgeh** [reply]
I used the stable release 0.5.4.

Awesome! It worked! Thanks!
